app/views/statements.scala.html

```diff
@@ -56,7 +56,10 @@ <h2 id="@tmwy.taxYear-transactions" class="heading-medium">@messages("statements
         <h2 id="@tmwy.taxYear-still-to-pay" class="heading-small">@messages("statements.stillToPay", tmwy.model.outstandingAmount.get.toCurrencyString)</h2>
         <p id="@tmwy.taxYear-due-by">@messages("statements.dueBy", tmwy.model.charges.head.dueDate.get.toLongDate, s"${tmwy.taxYear}")
             @if(appConfig.features.paymentEnabled()){
-                @messages("statements.dueBy.youCan") <a id="@tmwy.taxYear-payment-link" data-journey-click="StatementsPage:ClickLink:Pay" href="@controllers.routes.PaymentController.paymentHandoff(tmwy.model.outstandingAmount.get.toPence).url">@messages("statements.dueBy.link")</a>.
+                @messages("statements.dueBy.youCan") <a id="@tmwy.taxYear-payment-link"
+                                                        data-journey-click="StatementsPage:ClickLink:Pay"
+                                                        aria-label="@messages("statements.dueBy.screenReaderLink")"
```

Comment:
Should this be - 
`aria-label="@messages("statements.dueBy.screenReaderLink", s"${tmwy.taxYear - 1}", s"${tmwy.taxYear}")"`

Reply:
Ditto :)